### PR TITLE
fix-missing-token

### LINKED ISSUE #28: regression: 'sloppy version' should not require login
Very likely introduced in #21.

### PR DESCRIPTION
**Pull request by malud**
This will fix #28 issue with an early out

### CODE REVIEW

**Comment by jzaefferer** [comment]
Nice, thanks. Asked @var2dan to test this once.

**Comment by var2dan** [comment]
When I try to run sloppy version, I get no output
```
vardan@ubuntu:~/sloppy/cli/bundles/1.9.0$ ./sloppy-linux-amd64 version

vardan@ubuntu:~/sloppy/cli/bundles/1.9.0$ 
```

**Comment by malud** [comment]
```
➜  cli git:(feature/fix-missing-token) ✗ git rev-parse --short HEAD
19c54e6
➜  cli git:(feature/fix-missing-token) ✗ go build -o cli ./cmd
➜  cli git:(feature/fix-missing-token) ✗ ./cli version
A newer version of this runtime is available
Server has version: 1.9.0
User has version:

Check https://sloppy.io/knowledge-base/install-cli/ for install/update instructions
and https://github.com/sloppyio/cli/blob/master/CHANGELOG.md for the changelog
```

**Comment by jzaefferer** [comment]
`make beta` has the same issue, tested on OSX. `./sloppy-darwin-amd64 version` outputs nothing.

**Comment by jzaefferer** [comment]
Mostly unrelated, I can't build the linux binary anymore:
```
Start compiling sloppy 1.9.0 for linux-amd64
# github.com/sloppyio/cli/cmd
2018/01/22 11:20:54 readSym out of sync
make: *** [cross] Error 2
```
Any idea what might be causing that?

**Comment by jzaefferer** [comment]
Testing with Go 1.9 now

**Comment by jzaefferer** [comment]
Unfortunately that has the same issue. Still no output.

File: cmd/cli.go [hunk 1]
```diff
@@ -56,7 +48,8 @@ func main() {
 	// Update mechanism
 	update := make(chan struct{}, 1)
 	if len(args) > 0 && args[0] == "version" {
-		update <- struct{}{}
+		fmt.Println(command.Version)
```
Comment (malud) [inline cmd/cli.go]: This variable gets set during the build process `scripts/compile.sh:40`. At this point we don't have to ask the backend, the current build version is known here.

**Pull request merged**